apix-impl/src/main/java/eu/xenit/apix/alfresco/comments/CommentService.java

```diff
@@ -96,9 +96,10 @@ protected Comment toComment(org.alfresco.service.cmr.repository.NodeRef document
         String content;
         try {
             content = IOUtils.toString(contentService.getContent(apixCommentNodeRef).getInputStream());
-        } catch (IOException ioException) {
-            log.error("Error reading content for comment {}", commentNodeRef, ioException);
-            content = "Comment not available: error reading content";
+        } catch (IOException e) {
+            String message = String.format("Encountered an IOException while handling comment %s", commentNodeRef);
+            log.error(message, e);
+            throw new AlfrescoRuntimeException(message, e);
```

Comment:
If we throw an exception, there is no need to log it. It will also be logged when it's not caught higher up, leading to 2 loglines for the same error.